resources/js/utils/helpers.ts

```diff
@@ -1,6 +1,6 @@
 export const config = (key: string) => window?.cfg?.[key];
 
 export const asset = (uri: string) => {
-  const assetsUrl = config('assetsUrl') ?? (window as any).assetUrl;
+  const assetsUrl = config('assetsUrl') ?? (window as unknown as Record<string, unknown>).assetUrl;
```

Comment:
Sorry for the confusion.

By defining `assetUrl` in _global.d.ts_ as belonging to the global (window) module, we've communicated to TypeScript that `assetUrl` is type-safe and valid.

Given that, you should be able to do:

```ts
const assetsUrl = config('assetsUrl') ?? window.assetUrl;
```

TypeScript will recognize that `window.assetUrl` is expected to be a `string` value rather than being confused and thinking we've made a typo.